content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+
+{{< codenew file="admin/resource/memory-constraints-pod-3.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-3.yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Kontainer menentukan permintaan memori yang
+terlalu kecil:
+
+```
+Error from server (Forbidden): error when creating "examples/admin/resource/memory-constraints-pod-3.yaml":
+pods "constraints-mem-demo-3" is forbidden: minimum memory usage per Container is 500Mi, but request is 100Mi.
+```
+
+## Membuat Pod yang tidak menentukan permintaan ataupun limit memori
+
+
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut tidak menentukan
+permintaan memori dan juga limit memori.
+
+{{< codenew file="admin/resource/memory-constraints-pod-4.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-4.yaml --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail tentang Pod:
+
+```
+kubectl get pod constraints-mem-demo-4 --namespace=constraints-mem-example --output=yaml
+```
+
+Keluaran yang dihasilkan menunjukkan Kontainer pada Pod memiliki permintaan memori 1 GiB dan limit memori 1 GiB.
+Bagaimana Kontainer mendapatkan nilai tersebut?
+
+```
+resources:
+  limits:
+    memory: 1Gi
+  requests:
+    memory: 1Gi
+```
+
+Karena Containermu tidak menentukan permintaan dan limit memori, Kontainer tersebut diberikan
+[permintaan dan limit memori semula](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)
+dari LimitRange.
+
+Pada tahap ini, Kontainermu mungkin saja berjalan ataupun mungkin juga tidak berjalan. Ingat bahwa prasyarat
+untuk tugas ini adalah Node harus memiliki setidaknya 1 GiB memori. Jika tiap Node hanya memiliki
+1 GiB memori, maka tidak akan ada cukup memori untuk dialokasikan pada setiap Node untuk memenuhi permintaan 1 Gib memori. Jika ternyata kamu menggunakan Node dengan 2 GiB memori, maka kamu mungkin memiliki cukup ruang untuk memenuhi permintaan 1 GiB tersebut.
+
+Menghapus Pod:
+
+```
+kubectl delete pod constraints-mem-demo-4 --namespace=constraints-mem-example
+```
+
+## Pelaksanaan batasan minimum dan maksimum memori
+
+Batasan maksimum dan minimum memori yang yang ditetapkan pada sebuah Namespace oleh LimitRange dilaksanakan
+hanya ketika Pod dibuat atau diperbarui. Jika kamu mengubah LimitRange, hal tersebut tidak akan memengaruhi
+Pods yang telah dibuat sebelumnya.
+
+## Motivasi untuk batasan minimum dan maksimum memori
+
+Sebagai seorang administrator klaster, kamu mungkin ingin menetapkan pembatasan jumlah memori yang dapat digunakan oleh Pod.
+Sebagai contoh:
+
+* Tiap Node dalam sebuah klaster memiliki 2 GB memori. Kami tidak ingin menerima Pod yang meminta
+lebih dari 2 GB memori, karena tidak ada Node pada klater yang dapat memenuhi permintaan tersebut.
```

Comment:
typo?
```suggestion
lebih dari 2 GB memori, karena tidak ada Node pada klaster yang dapat memenuhi permintaan tersebut.
```

Reply:
Sorry. Fixed in e323b8c5920d6ae02c050306495e784b8e19ad9f